tasks/agent-linux.yml

```
@@ -6,6 +6,12 @@
     agent_datadog_before_7400: "{{ agent_datadog_major is defined and agent_datadog_minor is defined and agent_datadog_major
       | int < 8 and agent_datadog_minor | int < 40 }}"
 
+- name: Set before 6/7.61.0 flag
+  set_fact:
+    datadog_before_7610: "{{ datadog_major is defined and datadog_minor is defined and datadog_bugfix is defined
```

```suggestion
    agent_datadog_before_7610: "{{ datadog_major is defined and datadog_minor is defined and datadog_bugfix is defined
```
renaming for consistency with other `set before xxxx flag` jobs, and since `agent_datadog_before_7610` is used on line 83